macros.in

```diff
@@ -1352,5 +1352,16 @@ end
     end
 }
 
+# buildsystem defaults
+%buildsystem_default_prep() %autosetup -p1
+
+# example buildsystem macros for autotools
```

Comment:
Is autotools really so prominent to be part of defaults?

Reply:
"Defaults" seems ambiguous in this context of %buildsystem_default_*. But assuming you mean whether autotools is prominent enough to have macros for it shipped with rpm itself... it really is, isn't it? Also, we're carrying %configure and all that, it'd seem strange to not to have the boilerplate-free counterparts too. As the sole buildsystem shipped with rpm it does seem strange, but as discussed elsewhere, we need to ship a cmake buildsystem too.

Reply:
I just wanted to open the question, because I am maintaining mainly rubygem-* packages where autotools are not used and RPM itself moved from autotools to cmake. Juste bing devils advocate, nothing more :smiling_imp:

Reply:
Ack. At any rate, a PR like this needs to have *something* for demonstration purposes, and because we have all that other stuff for autotools, it's the lowest bar.